quickfixj-core/src/main/java/quickfix/mina/initiator/IoSessionInitiator.java

```diff
@@ -261,12 +261,13 @@ private void handleConnectException(Throwable e) {
                 e = e.getCause();
             }
             final String nextRetryMsg = " (Next retry in " + computeNextRetryConnectDelay() + " milliseconds)";
```

Comment:
Was there a specific reason why you decided against your first idea of adding the `socketAddress` as argument?

Reply:
Do you mean to add `socketAddress` as `quickfix.SessionStateListener#onConnectException` method argument?
It was not my firs idea actually, I proposed to wrap the exception and add some details to this exception

I'm not sure if this is a good approach to change the interface method and its signature, because it will force all the clients to modify their code when it may be not necessary

That's why I just extended the exception details, so after update to new version nobody will need to modify their existing code